src/v/datalake/translation_task.cc

```diff
@@ -278,6 +278,15 @@ translation_task::finish(
           fmt::join(write_result.data_files, ", "));
     }
 
+    size_t rows_added = 0;
+    size_t bytes_added = 0;
+    for (const auto& file : write_result.data_files) {
+        rows_added += file.local_file.row_count;
+        bytes_added += file.local_file.size_bytes;
+    }
+    _translation_probe->on_translation_finished(
+      write_result.data_files.size(), rows_added, bytes_added);
```

Comment:
Are we explicitly not counting DLQ files here? We do have the metric for invalid records so we're covered in that regard, but thought I'd ask in case we want to expose these details for them as well.

Reply:
Yeah, let's add a separate counter for dlq files. My thought process is that dlq table has more predictable schema and partitioning, so the risk of file count explosion is low.